.browserslistrc

```diff
@@ -0,0 +1,2 @@
+> 1%
+last 2 versions
```

Comment:
Supporting the last 2 versions of every browser means supporting

> Internet Explorer Mobile (0.23% market share globally)
> Blackberry Browser (0.07%)
> Opera Mobile (0.01%)
> QQ Browser (0%)
> Baidu Browser (0%)

via https://jamie.build/last-2-versions

I think we should stick with the `> 1%` line